src/main/java/com/minecolonies/coremod/network/messages/BuildToolPasteMessage.java

```diff
@@ -50,14 +49,14 @@
      */
     private static final int SUPPLY_SHIP_CHEST_HEIGHT = 6;
 
-    private boolean complete;
-    private String                   structureName;
-    private String                   workOrderName;
-    private int                      rotation;
-    private BlockPos                 pos;
-    private boolean                  isHut;
-    private boolean                  mirror;
-    private WindowBuildTool.FreeMode freeMode;
+    private boolean                                                     complete;
+    private String                                                      structureName;
+    private String                                                      workOrderName;
+    private int                                                         rotation;
+    private BlockPos                                                    pos;
+    private boolean                                                     isHut;
+    private boolean                                                     mirror;
+    private com.structurize.coremod.client.gui.WindowBuildTool.FreeMode freeMode;
```

Comment:
is that impossible to import {}; or just forgotten?

Reply:
On that, it's impossible, because it has the same name as a class in this repo.